tweets/2020/update-authors-app.tweet

```diff
@@ -0,0 +1 @@
+🤖📯 Built with Probot https://mobile.twitter.com/gr2m/status/1277723825430360064
```

Comment:
Remove `mobile` subdomain? Not sure if it matters.

```suggestion
🤖📯 Built with Probot https://twitter.com/gr2m/status/1277723825430360064
```

Reply:
thanks :)